src/index.html

```diff
@@ -2,16 +2,196 @@
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      name="form"
+    >
+      <fieldset class="box_style">
+        <legend>Personal information</legend>
+
+        <div class="field_wrapper">
+          <label for="surname">Surname: </label>
+
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            value=" "
+            minlength="3"
+            maxlength="100"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="field_wrapper">
+          <label for="name">Name: </label>
+
+          <input
+            type="text"
+            name="name"
+            id="name"
+            value=" "
+            minlength="3"
+            maxlength="100"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="field_wrapper">
+          <label for="age">How old are You? </label>
+
+          <input
+            type="number"
+            name="age"
+            id="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+
+        <div class="field_wrapper">
+          <label for="date">Full date of birth: </label>
+
+          <input
+            type="date"
+            name="date"
+            id="date"
+            value=" "
+          >
+        </div>
+
+        <label for="check">I accept the term of the agreement </label>
+
+        <input
+          type="checkbox"
+          value="check"
+          name="check"
+          id="check"
+        >
+      </fieldset>
+
+      <fieldset class="box_style">
+        <legend>Registration</legend>
+
+        <div class="field_wrapper">
+          <label for="email">E-mail: </label>
+
+          <input
+            type="email"
+            name="email"
+            id="email"
+            value=" "
+            placeholder="email@example.com"
+            multiple
+          >
+        </div>
+
+        <label for="user-password">Password: </label>
+
+        <input
+          name="user-password"
+          id="user-password"
+          type="password"
+          required autocomplete="off"
+        >
+      </fieldset>
+
+      <fieldset class="box_style">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="field_wrapper">
+          <label for="cat">Do you love cats? </label>
+
+          <input
+            type="radio"
+            id="yes"
+            name="cat"
+            value="Y"
+          >
+          <label for="yes">Yes</label>
+
+          <input
+            type="radio"
+            id="no"
+            name="cat"
+            value="N"
+          >
+          <label for="no">No</label>
+        </div>
+
+        <div class="field_wrapper">
+          <label for="color">What is your favorite color? </label>
+
+          <input
+            type="color"
+            name="color"
+            value=" "
```

Comment:
Empty space as a default value in input is not a good practice. Please remove this attribute `value=" "` it's redundant here